firefox-ios/Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -145,20 +151,29 @@ class ContentBlockerSettingViewController: SettingsTableViewController {
         guard let defaultFooter = _defaultFooter else { return nil }
 
         if section == 0 {
-            // TODO: Get a dedicated string for this.
             let title: String = .TrackerProtectionLearnMore
 
             let theme = themeManager.currentTheme(for: windowUUID)
-            let font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .subheadline, size: 12.0)
+            let font = FXFontStyles.Regular.caption1.scaledFont()
             var attributes = [NSAttributedString.Key: AnyObject]()
             attributes[NSAttributedString.Key.foregroundColor] = theme.colors.actionPrimary
             attributes[NSAttributedString.Key.font] = font
 
-            button.setAttributedTitle(NSAttributedString(string: title, attributes: attributes), for: .normal)
-            button.addTarget(self, action: #selector(moreInfoTapped), for: .touchUpInside)
-            button.isHidden = false
+            linkButton.setAttributedTitle(NSAttributedString(string: title, attributes: attributes), for: .normal)
```

Comment:
thanks for this update!! Can we also use `LinkButtonViewModel` to configure the button? We want the new link buttons to have an underline style.

Reply:
Hi @cyndichin, the button is now configured using `LinkButtonViewModel` and I also updated the screenshots. Thank you 🙏